sqlglot/parser.py

```diff
@@ -2760,7 +2760,7 @@ def _parse_user_defined_function(
         if not self._match(TokenType.L_PAREN):
             return this
 
-        expressions = self._parse_csv(self._parse_udf_kwarg)
+        expressions = self._parse_csv(lambda: self._parse_column_def(self._parse_id_var()))
```

Comment:
maybe. make this a function? so it’s not done in tsql too

Reply:
sure